audition/selection_rules.py

```diff
@@ -1,6 +1,5 @@
-
-def highest_metric_value(df, train_end_time, metric, param):
-    """Pick the model group with the highest current metric value
+def best_metric_value(df, train_end_time, metric, param):
```

Comment:
I'd suggest `best_current_value` rather than `best_metric_value` just to be a little more clear.